app/routes/analysis.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
```

Comment:
did you mean to merge this in now? Probably should wait until it's actually working code, I'll just copy into a new a branch when I work on it